pkg/controllers/machinesync/machine_sync_controller_test.go

```diff
@@ -429,18 +462,32 @@ var _ = Describe("With a running MachineSync Reconciler", func() {
 				Eventually(k.UpdateStatus(mapiMachine, func() {
 					mapiMachine.Status.AuthoritativeAPI = mapiv1beta1.MachineAuthorityMachineAPI
 				})).Should(Succeed())
+
+				capiMachine = capiMachineBuilder.WithName(mapiMachine.Name).Build()
 			})
 
 			It("should successfully create the CAPI machine", func() {
-				Eventually(k.Get(
-					clusterv1resourcebuilder.Machine().WithName(mapiMachine.Name).WithNamespace(capiNamespace.Name).Build(),
-				)).Should(Succeed())
+				Eventually(k.Get(capiMachine)).Should(Succeed())
 			})
```

Comment:
I don't think this is testing anything. We already asserted that Create() succeeded in BeforeEach. Are we wondering if it was deleted since? I would just delete this test.